dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1118,14 +1155,19 @@ export INVOKED
 case "$ACTION" in
 
   # Start the server in the foreground
-  run | debug)
+  run | debug | checkpoint)
+
+    if restoreServer false
```

Comment:
I don't get this syntax. Waht is the trailing `false` in
`if restoreServer false`